lib/internal/child_process.js

```diff
@@ -375,12 +375,11 @@ ChildProcess.prototype.spawn = function(options) {
       err === UV_ENFILE ||
       err === UV_ENOENT) {
     process.nextTick(onErrorNT, this, err);
+
     // There is no point in continuing when we've hit EMFILE or ENFILE
     // because we won't be able to set up the stdio file descriptors.
-    // It's kind of silly that the de facto spec for ENOENT (the test suite)
-    // mandates that stdio _is_ set up, even if there is no process on the
-    // receiving end, but it is what it is.
-    if (err !== UV_ENOENT) return err;
```

Comment:
Oh hey, it's that comment's fifth birthday tomorrow - that's long enough for me to change my mind. :-)